### PR TITLE
Would it be possible to add async style

### PR DESCRIPTION
**Pull request by iwin2471**
I added async style (#5 )
Would it be possible to add async style

### CODE REVIEW

**Comment by iwin2471** [comment]
sry I signed it!

File: calendar/quickstart/index.js [hunk 1]
```diff
@@ -64,16 +68,19 @@ function getAccessToken(oAuth2Client, callback) {
...
-      fs.writeFile(TOKEN_PATH, JSON.stringify(token), (err) => {
-        if (err) console.error(err);
-        console.log('Token stored to', TOKEN_PATH);
-      });
+      try{
+        fs.writeFileSync(TOKEN_PATH, JSON.stringify(token));
+      }catch(err){
+        console.error(err);
+      }
+      console.log('Token stored to', TOKEN_PATH);
```
Comment (grant) [inline calendar/quickstart/index.js]: Only log if it worked.

Comment (iwin2471) [reply calendar/quickstart/index.js]: oh ok I will fix it

**Comment by iwin2471** [comment]
sorry for my short english sometimes I have grammer mistakes...

**Comment by iwin2471** [comment]
well I fixed it check my code plz

File: calendar/quickstart/index.js [hunk 2]
```diff
@@ -64,16 +68,19 @@ function getAccessToken(oAuth2Client, callback) {
     input: process.stdin,
     output: process.stdout,
   });
+
   rl.question('Enter the code from that page here: ', (code) => {
     rl.close();
-    oAuth2Client.getToken(code, (err, token) => {
+      oAuth2Client.getToken(code, (err, token) => {
```
Comment (grant) [inline calendar/quickstart/index.js]: This spacing is incorrect.

File: calendar/quickstart/index.js [hunk 3]
```diff
@@ -18,17 +18,19 @@
 const fs = require('fs');
 const mkdirp = require('mkdirp');
 const readline = require('readline');
-const google = require('googleapis');
+const {google} = require('googleapis');
 const OAuth2Client = google.auth.OAuth2;
 const SCOPES = ['https://www.googleapis.com/auth/calendar.readonly'];
 const TOKEN_PATH = 'credentials.json';
+const token = {};
```
Comment (grant) [inline calendar/quickstart/index.js]: This variable doesn't need to be file-wide, put it next to the other code (l45):

`const token = {};`

File: calendar/quickstart/index.js [hunk 4]
```diff
@@ -64,16 +68,19 @@ function getAccessToken(oAuth2Client, callback) {
     input: process.stdin,
     output: process.stdout,
   });
+
```
Comment (grant) [inline calendar/quickstart/index.js]: Remove this newline.

**Pull request merged**